bench-tps/src/cli.rs

```diff
@@ -496,22 +529,133 @@ pub fn extract_args(matches: &ArgMatches) -> Config {
             .value_of("instruction_padding_program_id")
             .map(|target_str| target_str.parse().unwrap())
             .unwrap_or_else(|| FromOtherSolana::from(spl_instruction_padding::ID));
+        let parsed_data_size;
+        assign_or_return_err!(
+            parsed_data_size,
+            data_size.to_string().parse::<u32>(),
+            "Can't parse padded instruction data size"
+        );
         args.instruction_padding_config = Some(InstructionPaddingConfig {
             program_id,
-            data_size: data_size
-                .to_string()
-                .parse()
-                .expect("Can't parse padded instruction data size"),
+            data_size: parsed_data_size,
         });
     }
 
     if let Some(num_conflict_groups) = matches.value_of("num_conflict_groups") {
-        args.num_conflict_groups = Some(
-            num_conflict_groups
-                .parse()
-                .expect("Can't parse conflict groups"),
+        let parsed_num_conflict_groups;
+        assign_or_return_err!(
+            parsed_num_conflict_groups,
+            num_conflict_groups.parse(),
+            "Can't parse conflict groups"
         );
+        args.num_conflict_groups = Some(parsed_num_conflict_groups);
     }
 
-    args
+    Ok(args)
+}
+
+mod tests {
+    use {
+        super::*,
+        solana_sdk::signature::{read_keypair_file, write_keypair_file, Keypair, Signer},
```

Comment:
Also, since you are using `super::*`, you don't need to reimport elements imported in the top-level module.
```suggestion
        solana_sdk::signature::{write_keypair_file, Signer},
```

Reply:
*The evil of the wildcard imports is raising its ugly head*

Reply:
Could you clarify what exactly is inconsistent?
After using `super::*` it looks similar to the other tests that use `super::*`.

Reply:
But this one isn't using `super::*`

Reply:
Lol, yeah, it's weird it's not showing that preview as outdated.
Oh well. I'm not going to fight on this one anymore anyway.